software/ADPL_electron/ADPL_electron.ino

```diff
@@ -72,6 +73,9 @@ void setup() {
     attachInterrupt(UP, up_pushed, FALLING);
     attachInterrupt(DOWN, down_pushed, FALLING);
     attachInterrupt(RESET, res_pushed, FALLING);
+
+    //initialize pinch valve time attribute
+    pinchValve.wait_time = 0;
```

Comment:
It represents the last time the pinch valve was moved. I have changed it's name to lastTime, which is consistent with the syntax of the bucket (which has a similar variable)